atoms/Table/style.scss

```diff
@@ -0,0 +1,24 @@
+.Table {
```

Comment:
I think it would be best if you prefix the classnames with `hui-` to be consistent with other components in this repo like this: https://github.com/everydayhero/hui/blob/master/buttons/Button/index.js#L80